package.json

```
@@ -13,9 +13,9 @@
     "networks-inject-dep": "node src/inject_network_info_deps.js",
     "networks-reset": "truffle networks --clean && npm run networks-inject",
     "networks": "truffle networks",
-    "compile-todo": "eslint --fix . && eslint . && truffle compile",
-    "compile": "truffle compile",
-    "restore": "truffle compile --all && npm run networks-reset",
+    "compile-todo": "eslint --fix . && eslint . && yarn compile",
+    "compile": "truffle compile -all && truffle compile --all --config truffle.4.js",
```

Is there a missing `-`?

```suggestion
    "compile": "truffle compile --all && truffle compile --all --config truffle.4.js",
```